hud-data/7hud.json

```diff
@@ -6,6 +6,44 @@
     "steam_group": "7HUD",
     "album": "https://imgur.com/a/yluqH"
   },
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "crosshairs",
+    "materials",
+    "customization"
+  ],
+  "traits": {
+    "os": [
+      "windows",
+      "mac",
+      "linux"
+    ],
+    "gamemode": [
+      "tournament",
+      "comp",
+      "arena",
+      "ad",
+      "ctf",
+      "cp",
+      "dom",
```

Comment:
dom is not a gamemode, plesae remove it from all the traits

Reply:
Done. I did doubt it, but [TF2Maps](https://tf2maps.net/tags/domination/) had the tag, so I left it in.

Reply:
Yeah it's a gamemode in terms of map logic and such, but it doesn't affect HUDs.